dev/release/post-08-docs.sh

```diff
@@ -84,6 +84,18 @@ if [ "$is_major_release" = "yes" ] ; then
   previous_series=${previous_version%.*}
   mv docs_temp docs/${previous_series}
 fi
+# Update DOCUMENTATION_OPTIONS.theme_switcher_version_match and
```

Comment:
I would keep doing a single commit for updating the documentation and another for the banner, in case a review is necessary is much easier to do.

```suggestion
git add docs
git commit -m "[Website] Update documentations for ${version}"

# Update DOCUMENTATION_OPTIONS.theme_switcher_version_match and
```